src/examples/calls.ml

```diff
@@ -32,8 +32,8 @@ let main ()
   (* L2: the call here is only valid after L1*)
   call_ret swap
 
-let main1_false ()
-(*@ Norm(emp, 1) @*)
+let main1 ()
```

Comment:
I removed `_false` and moved this into the test file since this example works now